stage1/rootfs/usr_from_host/Makefile

```diff
@@ -0,0 +1,15 @@
+# do not include systemd in stage1, let's find it at run-time
+
+usr.done install: Makefile
+	echo mkdir -p "\$${ROOT}" > install.tmp
+	echo ln -sf usr-from-host \"\$${ROOT}/flavor\" >> install.tmp
+	mv install.tmp install
+	cp -f install ../aggregate/install.d/00usr
+	touch usr.done
+
+.PHONY: clean distclean
```

Comment:
The `.PHONY` line implicitely creates a distclean target.

I added one explicitely in the same way as `stage1/rootfs/usr_from_src/Makefile`.